### PR TITLE
Survey result Graph

### PR DESCRIPTION
**Pull request by Pyae24**
## Hello ama
- This PR is for the issue https://github.com/khinezarthwe/mmdt_web_app/issues/73
### Features

- I added a button to display the survey results in the survey page.

- It will redirect to a new URL path and display the user responses to survey forms.

- The results (Multiple Choice, Checkbox, Dropdown Question types) can be displayed as a Bar chart or Pie chart selected by the admin.

### Issues to fix

- Currently, an admin can choose only two charts for the question types of multiple choice, dropdown, and checkbox.
- The results of Text and Long Text question types are displayed back the same as users' inputs with bullet points.
- Admin's ability to select the chart type should be deleted for the question types of Text, Long Text, and Sliding Scale as they have only one type of data visualization.

### CODE REVIEW

File: mmdt/templates/survey/all_results.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{% extends "base.html" %}
+{% load static %}
+{% load templatetags %}
+{% block content %}
+<h2>Survey Results</h2>
+<script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
```
Comment (khinezar-mercari) [inline mmdt/templates/survey/all_results.html:6]: Don't use cdn, Please use local js that uploaded by Than Myo Htet.

File: mmdt/survey/migrations/0004_question_chart_type.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    dependencies = [
+        ('survey', '0003_alter_question_pub_date'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='question',
+            name='chart_type',
+            field=models.CharField(blank=True, choices=[('PC', 'Pie Chart'), ('BC', 'Bar Chart')], default='PC', max_length=2, null=True),
```
Comment (khinezar-mercari) [inline mmdt/survey/migrations/0004_question_chart_type.py:16]: Update DB diagram

**Comment by khinezar-mercari** [comment]
be sure survey result can only see when admin release the survey to all

**Comment by khinezarthwe** [comment]
Please put `is_result_release` in survey table and allow admin to handle it.
We will only show survey result button when admin set the TRUE to `is_result_release`

**Pull request merged**